src/App/DeliveryOptions/Service/DeliveryOptionsService.php

```diff
@@ -93,14 +94,20 @@ public function createAllCarrierSettings(PdkCart $cart): array
 
         [$packageType, $carriers] = $this->getValidCarrierOptions($cart);
 
+        $showPriceSurcharge =
+            Settings::get(CheckoutSettings::PRICE_TYPE, CheckoutSettings::ID) === CheckoutSettings::PRICE_TYPE_INCLUDED;
+
         $settings = [
-            'packageType'     => $packageType,
-            'carrierSettings' => [],
+            'packageType'           => $packageType,
+            'carrierSettings'       => [],
+            'basePrice'             => $cart->shipmentPrice,
+            'priceStandardDelivery' => $showPriceSurcharge ? $cart->shipmentPrice : 0,
```

Comment:
`showPriceSurcharge` is een root level setting op de delivery options. Je bent nu een hele functionaliteit aan het bouwen die bedoeld is om een simpele true/false toggle te zijn

Nu ik in de D.O. kijk doet die setting niet heel veel, misschien is die juist kapot.